packages/marketplace/src/components/pages/settings/developer-settings-organisation-tab/company-information.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react'
+import {
+  H3,
+  FlexContainerResponsive,
+  Content,
+  FlexContainerBasic,
+  GridItem,
+  Grid,
+  Button,
+  Formik,
+  Input,
+  ImageInput,
+  TextArea,
+  SelectBox,
+  Form,
+  LevelRight,
+} from '@reapit/elements'
+import styles from '@/styles/pages/developer-settings-organisation-tab.scss?mod'
```

Comment:
If you add mt-8 as per comment in the sass, you can remove this file

Reply:
Thanks. I have patched it